x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/file/FileUserPasswdStore.java

```diff
@@ -191,9 +193,13 @@ public void onFileDeleted(Path file) {
         @Override
         public void onFileChanged(Path file) {
             if (file.equals(FileUserPasswdStore.this.file)) {
-                logger.info("users file [{}] changed. updating users... )", file.toAbsolutePath());
+                Map<String, char[]> previousUsers = users;
```

Comment:
Would you mind making all these `previousXXX` local vars `final`? It's a small thing, but it helps when reading the code to know that compiler is enforcing it for us.